golem/task/tasksession.py

```diff
@@ -202,7 +202,9 @@ def send_verification_failure():
             )
 
         def verification_finished():
+            logger.debug("Verification finished handler.")
             if not self.task_manager.verify_subtask(subtask_id):
+                logger.debug("Verification failure: %s", subtask_id)

```

Comment:
https://github.com/golemfactory/golem-internal/wiki/Logging-guidelines#log-message-structure
```suggestion
                 logger.debug("Verification failure. subtask_id=%r", subtask_id)
```